src/mappings/merkleDistributor.ts

```diff
@@ -289,8 +295,11 @@ export function handleRewardsClaimed(event: RewardsClaimed): void {
     }
     claimedAmount.cumulativeClaimedAmount = cumulativeAmounts[i]
     claimedAmount.save()
+    unclaimedAmounts[i] = claim.cumulativeAmounts[i].minus(claimedAmount.cumulativeClaimedAmount)
```

Comment:
You always claim everything what is accumulated, so we can just set unclaimed amounts to empty array

Reply:
Ok, but array of zeros will be more consistent than just empty array.